src/Tests/NovaTestTrait.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace RonasIT\Support\Tests;
+
+use Illuminate\Contracts\Auth\Authenticatable;
+use Illuminate\Foundation\Testing\TestCase;
+use Illuminate\Support\Str;
+use Illuminate\Testing\TestResponse;
+
+trait NovaTestTrait
+{
+    protected function novaSearchParams(array $filters, string $search = '', int $perPage = 25): array
+    {
+        return [
+            'search' => $search,
+            'filters' => base64_encode(json_encode($filters)),
+            'perPage' => $perPage,
+        ];
+    }
+
+    protected function novaCreateResource(string $resourceClass, ?array $data = []): TestResponse
```

Comment:
let's add postfix to all api call helpers

```suggestion
    protected function novaCreateResourceAPICall(string $resourceClass, ?array $data = []): TestResponse
```